controllers/managedosversionchannel_controller.go

```diff
@@ -177,7 +177,10 @@ func (r *ManagedOSVersionChannelReconciler) reconcile(ctx context.Context, manag
 	readyCondition := meta.FindStatusCondition(managedOSVersionChannel.Status.Conditions, elementalv1.ReadyCondition)
 	if readyCondition == nil {
 		// First reconcile loop does not have a ready condition
-		return ctrl.Result{}, r.createSyncerPod(ctx, managedOSVersionChannel, sync)
+		if err := r.createSyncerPod(ctx, managedOSVersionChannel, sync); err != nil {
+			return ctrl.Result{RequeueAfter: time.Second}, fmt.Errorf("creating syncer pod: %w", err)
+		}
+		return ctrl.Result{}, nil
```

Comment:
After a closer look based on logs including this change I believe this changes has no real effect. On error the reconcile loop is always requeued, hence it was already requeued. Then the automatic requeue is eaten at line 184 so there aren't infinite requeues

Probably we could consider certain retries counter?